dist/Makefile

```diff
@@ -17,36 +17,74 @@ install: install_obsapisetup install_apache install_project_update install_logro
 
 install_obsapisetup: system_dirs
 	$(INSTALL) -m 755 setup-appliance.sh $(DESTDIR)$(OBS_BACKEND_PREFIX)/setup-appliance.sh
+	sed -i \
+	    -e 's|wwwrun\.www|$(APACHE_USER).$(APACHE_GROUP)|g' \
+	    -e 's|/srv/www/obs/api|$(OBS_API_PREFIX)|g' \
+	    -e 's|/srv/www/obs/overview|$(OBS_DOCUMENT_ROOT)/overview|g' \
+	    -e 's|/srv/obs|$(OBS_BACKEND_DATA_DIR)|g' \
+	    $(DESTDIR)$(OBS_BACKEND_PREFIX)/setup-appliance.sh
```

Comment:
setup-appliance.sh is a shell script and the values should be sourced from a config file or better detect the OS and set the according variables.

PLEASE SEE  ALSO https://github.com/M0ses/open-build-service/blob/fedora_packaging/dist/setup-appliance.sh#L603